### DISCUSSION TITLE
Fealpy installation issue

### DISCUSSION THREAD

**Discussion by tomyduby** [weihuayi/fealpy#47] [Installation]
Dear Wei Huayi,
     I am following instruction from READ.MD document for Windows. The first step,
``` python setup_win.py develop```
works fine. The installation fails with the second step:
```
(base) PS C:\tomy\software\python\fealpy> pip install https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/MeshPy-2018.2.1-cp37-cp37m-win_amd64.whl
Collecting MeshPy==2018.2.1
  ERROR: HTTP error 404 while getting https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/MeshPy-2018.2.1-cp37-cp37m-win_amd64.whl
  ERROR: Could not install requirement MeshPy==2018.2.1 from https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/MeshPy-2018.2.1-cp37-cp37m-win_amd64.whl because of error 404 Client Error: Not Found for url: https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/MeshPy-2018.2.1-cp37-cp37m-win_amd64.whl
ERROR: Could not install requirement MeshPy==2018.2.1 from https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/MeshPy-2018.2.1-cp37-cp37m-win_amd64.whl because of HTTP error 404 Client Error: Not Found for url: https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/MeshPy-2018.2.1-cp37-cp37m-win_amd64.whl for URL https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/MeshPy-2018.2.1-cp37-cp37m-win_amd64.whl
```
     Could you, please advise?
     Best Regards
     Tomy

**Comment by weihuayi** [comment]
Can you offer your version information of python?

**Comment by weihuayi** [comment]
I just check the website "https://download.lfd.uci.edu/pythonlibs/". Yes, we can not access it now. I will try to find another source for meshpy and pyamg.

**Comment by weihuayi** [comment]
Try to install the meshby by the instruction in https://anaconda.org/conda-forge/meshpy

**Comment by weihuayi** [comment]
Please check the new version of README.md.

**Comment by tomyduby** [comment]
Thanks @weihuayi : Problems resolved!
Now starts the more difficult task: learn to use you nice software. 
Best wishes :-)
Tomy

**Comment by weihuayi** [comment]
That's nice. The document in fealpy is still not good enough, and I am trying my best to working on it. If  you have any furthormore problem on using, please put it on here. We can discuss about it.
